components/filter/filter.js

```diff
@@ -443,6 +453,7 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		return html`
 			${searchResults}
 			<d2l-list
+				aria-live="polite" 
```

Comment:
Did we still want this, or did this cause the double reading bug in VoiceOver?

Reply:
Nope, I was removing that, sorry for the messy logs. Rebased instead of merging and I ended up messing up the branch 🤦